resources/github-actions.md

```diff
@@ -45,6 +128,21 @@ There is a link provided at the end of every C++ and Python build job where the
 
 ![](/assets/images/downloads.png)
 
+## Using Conda CI Artifacts Locally
+
+The artifacts that result from running `conda build` are conda channels. RAPIDS' CI system then compresses these conda channels into tarballs and uploads them to [https://downloads.rapids.ai/](https://downloads.rapids.ai/).
+
+The packages in the conda channel can be used by extracting the tarball to your local filesystem and using the resulting path in your conda commands.
```

Comment:
It's been 11 months since that was last discussed so it might be worth re-evaluating. I feel confident in `.conda` artifacts based on the experience I've had with conda-forge. We'll want to see if the artifact compression is any better.